Test for KHR_lights_punctual and BRDF normal distribution function

rsahlin opened the pull request:
The purpose of this model is to verify that the internal calculations does not add energy - and in this case especially by the normal distribution function used to calculate specular highlights.

Please see the README for more information.

rsahlin commented on 2.0/DirectionalLight/README.md:
```diff
@@ -0,0 +1,34 @@
+
+
+
+This means that the color of the specular highlight shall retain the same chromaticity as: RGB[0.9, 0.8, 0.1] * [0.6, 0.6, 0.6] = [0.54, 0.48, 0.06] = Chroma [0.48, 0.42, 0.09]
```
Hi @elalish  and thanks for your comments

Ooops, that is a mistake on my part. I was in a hurry and mixed up with the metal test.
I have updated the wording.

rsahlin commented on 2.0/DirectionalLight/README.md:
```diff
@@ -0,0 +1,34 @@
+The purpose of this test model is to verify that the BRDF and microfacet model used by a renderer preserves energy.  
+It is intended as a test to measure the amount of light factored in by the distribution functions (normal, shadowing and occlusion) - in this case especially the normal distribution function.  
+At least for realtime rasterizers it is common that the factor returned by this function is applied to the incoming light to calculate specular highlights.  
+In order to preserve energy it is important that this factor does not add light energy.  
```
Well, the main purpose of the white furnace test, as proposed in the below link, is to test the shadowing and masking function (of the microfacet model in your BRDF of choice)
This test is only meant to take the normal distribution function into consideration.

Link to proposal for white furnace test (2014)
https://www.jcgt.org/published/0003/02/03/paper.pdf

cx20 commented:
It is a minor issue, but it would be good to match the case of the folder names with the other sample folders.

|before|after|
|:------|:------|
|DirectionalLight/gl**tf**|DirectionalLight/gl**TF**|
|DirectionalLight/gl**tf**-Binary|DirectionalLight/gl**TF**-Binary|

Also, it might be nice to have a screenshot of the correct and the failure screenshots.

rsahlin commented:
Hi @cx20 and thanks for your comment!

Good catch - I'll update the names.
However, I'm a bit reluctant to upload a 'correct' screenshot as it may be taken as ground truth.
I'll think about it and include something. 
Perhaps the correct/failure result rendered using chroma.

rsahlin commented:
Updated model slightly, added two screenshots - one for what the output should resemble in the (s)RGB domain and one what it should look like when rendered in chroma.
For chroma.png the important thing is that the chroma does not change (drastically) in the specular highlight as this may indicate that energy is added - resulting in clipping of RGB values.

Does not seem like github notices that I have changed name of folders since I am on Windows - yay for case insensitive filenames :-)
